src/pages/signup/index.jsx

```diff
@@ -1,7 +1,174 @@
-import React from 'react';
+import React, { useState } from 'react';
+import bcrypt from 'bcryptjs';
+import isError from '../../utils/signUpFuncs';
 
 const SignUp = () => {
-  return <div>Hello Signup</div>;
+  const [email, setEmail] = useState('');
+  const [username, setUsername] = useState('');
+  const [phone, setPhone] = useState('');
+  const [password1, setPassword1] = useState('');
+  const [password2, setPassword2] = useState('');
```

Comment:
It is more recommend in many resources that form data should be handled in one state only

Reply:
Noted, I will update them accordingly